portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,7 +55,7 @@
    * which are stored in the messages variable.
    */
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {    
```

Comment:
nit: Looks like some extra whitespace got added to the end of this line.

Reply:
Done!